addons/frag/functions/fnc_dev_fragCalcDump.sqf

```diff
@@ -0,0 +1,61 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger, based on fnc_dev_debugAmmo by "ACE-Team"
+ * Dumps all ammo types to see if there's any reason to spawn fragments
+ * given hit power and distance. Good for grasping the values used in
+ * shouldFrag to cull non-fragementing rounds
+ *
+ * Arguments:
+ * 0: Display rounds that will never frag (power < 5).
+ *           Default value false <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [false, 10] call ace_frag_fnc_fragoCalcDump
+ *
+ * Public: No
+ */
+
+params [
+    ["_logAll", false, [false]]
+];
+
+
+private _allAmmoConfigs = configProperties [configFile >> "cfgAmmo", "isClass _x && !('ace_frag' in configName _x)", true];
+private _processedCfgAmmos = [];
+
+private _nPrinted = 0;
+
+diag_log text "//****************** fragCalcDump Beg ******************//";
+{ // Begin _allAmmoConfigs forEach
+    private _ammo = toLowerANSI configName _x;
+
+    if (_ammo == "" || {_ammo in _processedCfgAmmos} ) then { continue };
+
+
+    private _shouldFrag = [_ammo] call FUNC(shouldFrag);
+    if (_nSkip || _logAll) then {
+        private _fragInfo = [_ammo] call FUNC(getFragInfo);
+        _fragInfo params ["_fragRange", "_fragMaxVelocity", "", "_modifiedFragCount"];
+        private _fragCount = 4 * pi* _modifiedFragCount;
+        private _indirectHitRange = getNumber (configFile >> "cfgAmmo" >> _ammo >> "indirectHitRange");
+        private _indirectHit = getNumber (configFile >> "cfgAmmo" >> _ammo >> "indirectHit");
```

Comment:
```suggestion
        private _ammoConfig = configFile >> "CfgAmmo" >> _ammo;
        private _indirectHitRange = getNumber (_ammoConfig >> "indirectHitRange");
        private _indirectHit = getNumber (_ammoConfig >> "indirectHit");
```

Reply:
Didn't realize this was faster, implemented it throughout.